torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -307,21 +322,37 @@ AOTITorchError aoti_torch_create_tensor_from_blob(
     int32_t dtype,
     int32_t device_type,
     int32_t device_index,
-    AtenTensorHandle* ret_new_tensor) {
+    AtenTensorHandle* ret_new_tensor,
+    int8_t layout,
+    const uint8_t* serialized_md,
+    int64_t serialized_md_size) {
```

Comment:
perhaps we can name it in a more general way, e.g., `opaque_metadata` and `opaque_metadata_size`?

Reply:
Renamed the variables